packages/code-studio/src/styleguide/Inputs.tsx

```diff
@@ -188,15 +188,18 @@ function Inputs(): React.ReactElement {
               aria-labelledby="inputs-radios-heading"
               onChange={handleRadioChange}
               value={radioValue}
+              isInvalid={radioValue === '4'}
+              description="Select a radio item"
+              errorMessage={
+                radioValue === '4' ? 'Invalid radio selected' : undefined
+              }
```

Comment:
Actually I don't think you necessarily need a ternary here, as the error message is only shown when it's invalid anyway. But no matter.
```suggestion
              errorMessage="Invalid radio selected"
```